targets/zephyr/src/jerry-port.c

```diff
@@ -16,9 +16,13 @@
 #include <stdarg.h>
 
 #include <zephyr.h>
+#ifdef JERRY_DEBUGGER
+#include <sys/fcntl.h>
+#include <net/socket.h>
+#endif
 
 #include "jerryscript-port.h"
-
+#include "jrt.h"
```

Comment:
What is used from this header?
It's a private/internal header.

Reply:
Sounds reasonable.

Reply:
> JERRY_UNUSED

Hmm I guess it's not part of the "API"

Maybe just use `(void) variable_that_is_unused`?